tests/test-sources/plugins/lsp/_lsp.nix

```diff
@@ -265,4 +265,56 @@
           }
         ];
     };
+
+  package-fallback =
```

Comment:
I think we can set `test.buildNixvim` or `test.runNvim` to false here, unless building/running is actually part of the test? At first glance, it looks like we only care about the assertions, which will be checked even if `test.buildNixvim = false`.